### ISSUE TITLE
Preview window and Board show different graphics for the card

### ISSUE THREAD

**Issue by matthew-goddard** [frankk74/innovation#2]
Playing a game I have the card Agriculture played on my board. It shows a clock for the center icon but in the preview window in the top left it shows all Leaf icons. I believe these should be showing the same and all leafs?

![2022-07-28 10_01_23-WinInnovation AI](https://user-images.githubusercontent.com/52749806/181524947-da4f7f18-03df-4e69-bc24-a4e2e5f1b13e.png)

I came from your announcement article posted on BGG. Great work!

**Comment by frankk74** [comment]
Yes, thank you. I'm still trying to figure this bug out. It seems only the agriculture card suffers this weird rendering.

There are 2 of us working on this but I'm migrating the issues to this repo.

https://github.com/jsparkes/innovation/issues/2

Here are 2 other images of the same bug:

At start:

![agri-3-bug](https://user-images.githubusercontent.com/16901967/181669913-01c17318-470e-4717-b3a2-3fe96d41f061.png)

After mousing over:

![agri-2-bug](https://user-images.githubusercontent.com/16901967/181669939-de9a5d68-358b-4d80-9954-53b804a70fc0.png)
